examples/benchmark.rs

```diff
@@ -31,7 +31,7 @@ fn client(kind: Client) {
     let mut stream = TcpStream::connect(("192.168.69.1", port)).unwrap();
     let mut buffer = vec![0; 1_000_000];
 
-    let start = Instant::now();
+    let start = Instant::now().expect("Could not get the start time");
```

Comment:
Hang on. Why would this fail?

Reply:
It makes use of [SystemTime::duration_since] which returns a result type. Now that I'm looking at it, would it make more sense if `Instant::from_system_time` used [SystemTIme::elapsed] instead of [SystemTime::duration_since] the [UNIX_EPOCH]? Or would it make sense to have a `Instant::elapsed()`?

[SystemTime::duration_since]: https://doc.rust-lang.org/std/time/struct.SystemTime.html#method.duration_since
[SystemTime::elapsed]: https://doc.rust-lang.org/std/time/struct.SystemTime.html#method.elapsed
[UNIX_EPOCH]: https://doc.rust-lang.org/std/time/constant.UNIX_EPOCH.html

Reply:
It should just use `duration_since(UNIX_EPOCH).unwrap()`. I am not interested in handling cases where the time is less than UNIX epoch. The only such case is badly misconfigured (i.e. non-UTC timezone) systems without an RTC battery or ntpd.

Reply:
Actually, nevermind, not even that can happen because UNIX_EPOCH includes the timezone. So just unwrap it.

Reply:
:+1: